internal/config/cgmgr/cgmgr.go

```diff
@@ -79,8 +88,63 @@ func SetCgroupManager(cgroupManager string) (CgroupManager, error) {
 		}
 		return &systemdMgr, nil
 	case cgroupfsCgroupManager:
-		return new(CgroupfsManager), nil
+		cgroupsfsMgr := CgroupfsManager{
+			memoryPath:    cgroupMemoryPathV1,
+			memoryMaxFile: cgroupMemoryMaxFileV1,
+		}
+		if node.CgroupIsV2() {
+			cgroupsfsMgr.memoryPath = cgroupMemoryPathV2
+			cgroupsfsMgr.memoryMaxFile = cgroupMemoryMaxFileV2
+		}
```

Comment:
I don't think you need to branch here. I don't believe there's an expectation that we'd use cgroupfs on cgroupv2

 @giuseppe am I correct?

Reply:
Sure I can take that out for cgroupfs.

Reply:
cgroup v2 works both with systemd and cgroupfs